esprima.js

```diff
@@ -3664,7 +3673,13 @@
                 program.errors = extra.errors;
             }
         } catch (e) {
-            throw e;
+            console.log('Failed to parse ' + options.filename + ' Exception was: ' + e);
```

Comment:
so, if we fail to parse 1 file, we don't fail but instead we log a failure

I will actually change it to fail.